SplitImage/test/new.js

```diff
@@ -5,9 +5,43 @@
  
 describe ("SplitImage New Suite", function() {
   
+  // Get an HTML document defined by the pre-processor.
+  // This is a rough hack because HTML2JS seems to assign the
+  // key to the absolute URL, which is not reliable on
+  // continuous integration.
+  beforeEach(function() {
+    var keys = Object.keys(__html__);
```

Comment:
You can't use this hack anymore. You must create the DOM elements you need to test inside the javascript.